src/USER-LD/README

```diff
@@ -0,0 +1,7 @@
+USER-LD package
```

Comment:
This file would not be needed for contributing the pair style to USER-MISC, but a corresponding entry to the README file in that folder should be added.

Reply:
done